x-pack/plugins/security_solution/public/resolver/store/data/graphing.test.ts

```diff
@@ -215,7 +215,7 @@ describe('resolver graph with too much lineage', () => {
   beforeEach(() => {
     generator = new EndpointDocGenerator('seed');
     allEvents = generator.generateTree({ ancestors: 1, generations: 2, children: 2 }).allEvents;
-    childrenCursor = 'aValidChildursor';
```

Comment:
:information_source: Based on a cursory understanding of Latin roots, a "valid child ursor" could potentially be some kind of juvenile lycanthrope who takes the form of a bear?

Reply:
😂